server/mlabns/util/constants.py

```diff
@@ -40,8 +40,8 @@
 
 # URL to the privacy doc. All requests to http://mlab-ns.appspot.com/privacy
 # will be redirected to this URL.
-PRIVACY_DOC_URL = 'https://docs.google.com/a/google.com/document/d/1yQp7CcZngY6AfndoxvIbz8MzxcO7MpjZaj_VGFWe6Mo/pub'
+PRIVACY_DOC_URL = 'https://www.measurementlab.net/privacy/'
 
 # URL to the design doc. All requests to http://mlab-ns.appspot.com/docs will be
 # redirected to this URL.
-DESIGN_DOC_URL = 'https://docs.google.com/document/d/1eJhS75EZHDLmC6exggStr_b1euiR24_MVBJc1L6eH2c/view'
+DESIGN_DOC_URL = 'https://github.com/m-lab/mlab-ns/blob/master/DESIGN-DOC.md'
```

Comment:
Question: any reason to get rid of the Google Doc? I do think it's better to use github, but are then any concerns here that we aren't considering? cc @collina @pboothe

Reply:
URL gives me 404, so I don't know.

Reply:
This URL @pboothe ? 
https://docs.google.com/document/d/1eJhS75EZHDLmC6exggStr_b1euiR24_MVBJc1L6eH2c/view